scripts/murmur.ini

```diff
@@ -181,6 +195,26 @@ allowping=true
 ; (Note that you have to escape backslashes with \ )
 ;username=[-=\\w\\[\\]\\{\\}\\(\\)\\@\\|\\.]+
 
+; If a user has no stored channel (they've never been connected to the server
+; before, or rememberchannel is set to false) and the client hasn't been given
+; a URL that includes a channel path, the default behavior is that they will
+; end up in the root channel.
+;
+; You can set this setting to a channel ID, and the user will automatically be
+; moved into that channel instead. Note that this is the numeric ID of the
+; channel, which can be a little tricky to get (you'll either need to use an
+; RPC mechanism, watch the console of a debug client, or root around through
+; the Murmur Database to get it).
+;
+;; AFK channel
+;defaultchannel=
```

Comment:
I would remove the line `AFK channel` and set `defaultchannel=0` so that the user will directly seewhat kind of value this setting may have

Reply:
Done